rooms/templates/rooms/index.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+<div class="section flexbox center-items-horizontal full-height">
+  <div class="container">
+    <div class="row" id="react-root">
+      {% for r in rooms %}
+      <div class="col s6 m4">
+        <div class="card blue-grey darken-1">
+          <div class="card-content white-text">
+            <span class="card-title">{{ r }}</span>
+            <p>{{ r.description }}</p>
+            <p> ? / {{ r.capacity }}</p>
+          </div>
+        </div>
+      </div>
+      {% endfor %}
+    </div>
+  </div>
+</div>
+{% endblock %}
+
+{% block custom_scripts %}
+<script type="text/javascript" src="{% static 'bower_components/react/react.min.js' %}"></script>
+<script type="text/javascript" src="{% static 'bower_components/react/react-dom.min.js' %}"></script>
+<script type="text/javascript" src="{% static 'bower_components/babel-standalone/babel.min.js' %}"></script>
+<script type="text/jsx" src="{% static 'script/room.js' %}"></script>
+<script type="text/jsx">
+$(document).ready(function() {
+   ReactDOM.render(
```

Comment:
Simple placeholder until more complex React views come into shape - next PR will add them, hope I can keep this small shim till then?